allensdk/brain_observatory/ecephys/ecephys_session_api/ecephys_nwb_session_api.py

```diff
@@ -55,6 +57,22 @@ def __init__(self,
         self.additional_unit_metrics = additional_unit_metrics
         self.external_channel_columns = external_channel_columns
 
+        self._check_nwbfile_version()
+
+    def _check_nwbfile_version(self):
```

Comment:
this utility seems to be applicable to more than just ecephys. Why not move it up to a cross-project NWB utilities module?

Reply:
Moved!